src/modules/iframe/components/tools/DrawTool.js

```diff
@@ -228,7 +228,7 @@ export class DrawTool extends MvuElement {
 		const activeTool = this._getActiveTool(model);
 		const activeToolName = activeTool ? activeTool.name : 'noTool';
 		const removeAllowed = ['draw', 'modify'].includes(mode);
-		const unfinishedLine = mode === 'draw' && activeToolName === 'line' && validGeometry;
+		const unfinishedLine = mode === 'draw' && (activeToolName === 'polygon' || activeToolName === 'line') && validGeometry;
```

Comment:
similar to DrawToolContent: suggestion for a better readability
```suggestion
		const unfinishedLine = mode === 'draw' && ['polygon', 'line'].includes(activeToolName)&& validGeometry;
```